src/design/brand.stories.mdx

```diff
@@ -1,16 +1,14 @@
 import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
-// Importing a `.svg` directly, results in a failed build and importing
-// the `.svg.twig` file causes linting errors (since they are not tracked).
-// Therefore, we must track a twig file to satisfy CI.
-import brandLogo from './demo/logo.twig';
+import logoSrc from '../assets/brand/logo.svg';
```

Comment:
Rather than maintain a demo file just to display the image, I thought it made more sense to import the image source and display it via an `img` element in the story.